5 lesson

Chelt0n opened the pull request.

Chelt0n committed:
5 урок

robertBadamshin commented on app/src/main/java/com/example/materialdesignhomework/view/MarsPhotosFragment.kt:
```diff
@@ -2,19 +2,28 @@ package com.example.materialdesignhomework.view
 import com.example.materialdesignhomework.viewmodel.LatestImagesMarsViewModel
 
-class MarsPhotosFragment : Fragment(),MyOnClick {
+class MarsPhotosFragment : Fragment(), MyOnClick {
```
не делай так  MyOnClick

robertBadamshin commented on app/src/main/java/com/example/materialdesignhomework/view/MarsPhotosFragment.kt:
```diff
@@ -38,30 +47,48 @@ class MarsPhotosFragment : Fragment(),MyOnClick {
-                GridLayoutManager(context,4)
+                GridLayoutManager(context, 4)
             recyclerViewPhotosOfRovers.adapter = adapter
             adapter.setOnclickPhoto(this@MarsPhotosFragment)
```
почему бы тебе не избавиться от интерфейса и не передать просто лямбду?

robertBadamshin commented on app/src/main/java/com/example/materialdesignhomework/view/MarsPhotosFragment.kt:
```diff
@@ -38,30 +47,48 @@ class MarsPhotosFragment : Fragment(),MyOnClick {
+        binding.imageView.visibility = View.GONE
+    }
+
+    override fun click(nasa: Photo, position: Int) {
```
а позиция здесь зачем?

Chelt0n committed:
Merge remote-tracking branch 'origin/master' into 5_lesson

# Conflicts:
#	app/src/main/java/com/example/materialdesignhomework/view/MarsPhotosFragment.kt

Chelt0n committed:
5 урок анимация
при нажатии на картинку открываю ее на полный экран, еще один клик сворачивает ее
